src/app/3d/qgs3dmapcanvaswidget.cpp

```diff
@@ -153,6 +155,34 @@ Qgs3DMapCanvasWidget::Qgs3DMapCanvasWidget( const QString &name, bool isDocked )
   } );
   mOptionsMenu->addAction( mActionEnableEyeDome );
 
+  mActionSync2DNavTo3D = new QAction( tr( "Sync 2D Navigation to 3D Navigation" ) );
```

Comment:
Need to set the parent or the action will be leaked. Ownership of an action isn't transferred when it's added to a menu or toolbar.

```suggestion
  mActionSync2DNavTo3D = new QAction( tr( "2D Map View Follows 3D Camera" ), this );
```

Reply:
(trying to match the phrasing used elsewhere in QGIS as much as possible)